docs/guides/get-started-with-primitives.ipynb

```diff
@@ -7,15 +7,8 @@
    "source": [
     "# Get started with primitives\n",
     "\n",
-    "<LegacyContent>\n",
-    "<Admonition type=\"note\">\n",
-    "* This documentation is relevant to IBM Quantum&reg; Platform Classic. If you need the newer version, go to the new [IBM Quantum Platform documentation.](https://quantum.cloud.ibm.com/docs/guides/get-started-with-primitives)\n",
-    " * While this documentation uses the primitives from Qiskit Runtime, which allow you to use IBM&reg; backends, the primitives can be run on any provider by using the [backend primitives](#backend) instead.  Additionally, you can use the *reference* primitives to run on a local statevector simulator.  See [Exact simulation with Qiskit primitives](/docs/guides/simulate-with-qiskit-sdk-primitives) for details.\n",
-    "</Admonition>\n",
-    "</LegacyContent>\n",
     "<CloudContent>\n",
```

Comment:
This should remove CloudContent tag. It'd be good to grep for that in general.

You also don't need to use a list anymore